app/views/events/index.html.haml

```diff
@@ -1,23 +1,40 @@
 - title "Events"
 
-%h3 Upcoming Events:
-%table.events-list
-  - @events.days.each do |date, events|
-    %div.event-day
-      %div.event-date
-        %h1= l date, format: :long
-        %hr
+%div.events-heading
+  %h2.active= link_to 'Events', events_path
+  %h2= link_to 'Concerts', concerts_path
+
+- @events.days.each do |date, events|
+  %div.events-container
+    %div.events-date
+      %h3= l date, format: :long
+
+
+    %div.events-day
       %ul.events
       - events.each do |event|
         %li.event
-          %h3= event.title
-          %p= l event.beginning, format: :long
+          %h4= event.title
+          -if (l event.beginning, format: '%p') != ''
+            %p= l event.beginning, format: '%l:%M %p'
+          -else
+            %p All day
+          %p=link_to event.location, 'https://www.google.com/maps/search/' + event.location
           %p= event.description
+          - event.attachments.each do |a|
+            %a= link_to 'View Attachment', a['fileUrl'], alt: a['title'], target: "_blank"
```

Comment:
omg this is the same file as the other one 😂 Just the job for a layout! Check out what I did in [`/app/views/layouts/headline.html.erb`](https://github.com/cbothner/readback/blob/master/app/views/layouts/headline.html.erb) and try to do something similar for the header and footer. The event contents themselves would probably be a better fit for a partial in `/app/views/events/_event.html.haml`. If we implement `#to_partial_path` on `Event`, too, then you can just do
```haml
- events.each do |event|
  = render event
```
which is pretty pretty.